examples/gno.land/p/moul/cow/tree_test.gno

```diff
@@ -159,3 +161,232 @@ func TestTreeReverseIterateByOffset(t *testing.T) {
 		t.Errorf("Expected keys %v, got %v", expectedKeys, keys)
 	}
 }
+
+// Verify that Tree implements avl.ITree
+// var _ avl.ITree = (*Tree)(nil) // TODO: fix gnovm bug: ./examples/gno.land/p/moul/cow: test pkg: panic: gno.land/p/moul/cow/tree_test.gno:166:5: name avl not defined in fileset with files [node.gno tree.gno node_test.gno tree_test.gno]:
```

Comment:
It looks like it's fixed now. I tested it locally by importing ⁠`gno.land/p/demo/avl` for both the interface and the type ⁠`IterCbFn`, and the tests are running without any error.

Reply:
not for me :/

Reply:
@moul The changes in this commit, based on the current master, work fine on my machine (it compiles and the tests pass): https://github.com/aeddi/gno/commit/66b166b375cc677f0da7f34007995424f1a01a08

Maybe it's not what you want to do? But if so, I can open the PR.